### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by coderlol** [tim-group/java-statsd-client#37]
Just wondering if I need to fork it

**Comment by jonbartels** [comment]
If you have a specific question, ask it. If you're not satisfied with the response then use the power of opensource and just fork it!

I am not the project owner but I have gotten good response to issues and questions on this project.

**Comment by b-slim** [comment]
@jonbartels is there any reason that a good PR like [https://github.com/tim-group/java-statsd-client/pull/31] doesn't get any review ?

**Comment by jonbartels** [comment]
You're right in that the project is not very active. I know it can be frustrating to try and contribute and not have pull requests acknowledged.

I'd suggest three things to get your pull request accepted:
1) Update it so there is a description explaining what the change does and _why it is helpful or valuable to the project_. For example what use case does a batched UDP sender help solve?
2) Include test classes for your change.
3) Ask more direct questions of the project about the pull request. "Why wasn't this change reviewed or accepted?" is a clear question about the pull request. "Is this project dead?" 

For what its worth, I looked over your commit and it is well written. I wonder if using a different constructor (NonBlockingBatchStatsDClient?) instead of a boolean flag might be better. I am not sure though.

I hope that helps. Your contribution looks OK to me but I don't control the master project, I am just another contributor!

**Comment by waynr** [comment]
@coderlol I would recommend looking to DataDog/java-statsd-client for a more up-to-date codebase and a more responsive maintainer.
